### PR TITLE
New integrate-your-tech@es.md

### PR DESCRIPTION
**Pull request by SidVal**
`integrate-your-tech.md` in spanish :es: :+1:

### CODE REVIEW

File: content/docs/contribute/integrate-your-tech@es.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Si usted es un proveedor de tecnología para editores o anunciantes en la web, le invitamos a agregar soporte a AMP para que sus clientes puedan seguir aprovechando su tecnología y lograr nuestra visión conjunta para crear una mejor web.
+
+Existen 4 maneras principales de integrarse con AMP:
+
+## 1. Agregue su soporte a la extensión amp-analytics
+
+AMP Analytics le permite enviar eventos de vuelta a su servidor basado en disparadores configurados por usted. Hemos escrito una [guía de integración de análisis](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/integrating-analytics.md) para empezar.
+
+Si solo necesita agregar un píxel de seguimiento con parámetros dinámicos a su URL de seguimiento, eche un vistazo a [amp-pixel](/docs/reference/components/amp-pixel). Asegúrese de documentar el uso en sus páginas de soporte para los desarrolladores que deseen utilizar su tecnología con AMP.
```
Comment (bpaduch) [inline content/docs/contribute/integrate-your-tech@es.md]: Please change the link to: /docs/reference/components/amp-pixel.html

File: content/docs/contribute/integrate-your-tech@es.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Hay más de 90 proveedores de anuncios que han añadido soporte para funciones publicitarias relacionadas, como amp-ad. A continuación se muestra una [solicitud de extracción](https://github.com/ampproject/amphtml/pull/2299) de muestra de la red publicitaria [Criteo](https://github.com/ampproject/amphtml/blob/master/ads/criteo.md).
+
+## 3. Uso de la extensión amp-call-tracking
+
+Si proporciona servicios de medición de seguimiento de llamadas, su caso de uso puede ser compatible con la nueva extensión [amp-call-tracking](https://github.com/ampproject/amphtml/blob/master/extensions/amp-call-tracking/amp-call-tracking.md). Esta extensión reemplaza dinámicamente un número de teléfono en un hipervínculo para habilitar el seguimiento de llamadas, ejecutando una petición de CORS para sustituir el número.
```
Comment (bpaduch) [inline content/docs/contribute/integrate-your-tech@es.md]: Please change the link to: /docs/reference/components/amp-call-tracking.html

File: content/docs/contribute/integrate-your-tech@es.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Para obtener más información sobre cómo esta extensión podría funcionar para usted, consulte el sitio [AMP by Example](https://ampbyexample.com/components/amp-call-tracking/) o lea acerca de la [documentación](/docs/reference/components/amp-call-tracking).
```
Comment (bpaduch) [inline content/docs/contribute/integrate-your-tech@es.md]: please add `.html` to the last link

File: content/docs/contribute/integrate-your-tech@es.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## 4. Agregar una nueva extensión o incrustar
+
+Si su caso de uso no se puede acomodar mediante amp-analytics, amp-pixel o amp-ad, reporte un [problema en GitHub](https://github.com/ampproject/amphtml/issues/new) para discutir opciones alternativas. Damos la bienvenida a las nuevas extensiones que pueden ser ampliamente utilizados por un número de diferentes empresas. Consulte la [sección de componentes ampliados](https://github.com/ampproject/amphtml/blob/master/CONTRIBUTING.md#contributing-extended-components) para obtener más detalles.
+
+## 5. Usando el amp-iframe
+
+Se espera su uso solamente como último recurso. Si ninguno de las extensiones anteriores se ajustan a sus necesidades, podría utilizar la etiqueta genérica amp-iframe para permitir a los editores integrar su contenido, pero este enfoque viene con una serie de desventajas, para optimizar el rendimiento y la experiencia del usuario que puede leer [aquí](/docs/reference/components/amp-iframe#guideline:-prefer-specific-amp-components-to-amp-iframe).
```
Comment (bpaduch) [inline content/docs/contribute/integrate-your-tech@es.md]: Please change the link to `/docs/reference/components/amp-iframe.html#guideline:-prefer-specific-amp-components-to-amp-iframe`

**Comment by SidVal** [comment]
Ok, now? :+1:

**Pull request merged**